docs/kafka/kafka-instance-settings/README.adoc

```diff
@@ -0,0 +1,152 @@
+:service-url-connectors: https://console.redhat.com/application-services/connectors
+:getting-started-url-connectors: connectors/getting-started-connectors/README.adoc
+
+//OpenShift API Designer
+:product-long-api-designer: OpenShift API Designer
+:product-api-designer: API Designer
+:product-version-api-designer: 1
+:service-url-api-designer: https://console.redhat.com/application-services/api-designer/
+:getting-started-url-api-designer: api-designer/getting-started-api-designer/README.adoc
+
+//OpenShift API Management
+:product-long-api-management: OpenShift API Management
+:product-api-management: API Management
+:product-version-api-management: 1
+:service-url-api-management: https://console.redhat.com/application-services/api-management/
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-configuring-kafka-instance-settings"]
+= Configuring Kafka instance settings in {product-long-kafka}
+ifdef::context[:parent-context: {context}]
+:context: configuring-kafka-instance-settings
+
+// Purpose statement for the assembly
+[role="_abstract"]
+--
+As a developer of applications and services, you can review and modify settings for your Kafka instances. By modifying these settings, you can configure your Kafka instances to suit your particular environment.
+--
+
+
+[id="proc-editing-kafka-instance-settings_{context}"]
+== Reviewing and editing Kafka instance settings in {product-kafka}
+
+[role="_abstract"]
+Use the {product-long-kafka} web console to review and adjust the Kafka instance settings as needed.
+
+As an alternative to using the {product-kafka} web console, you can use the `rhoas` command-line interface (CLI) to update certain Kafka instance settings, as shown in the following example command:
+
+.Example CLI command to disable connection reauthentication
+[source]
+----
+rhoas kafka update --reauthentication false
+----
+
+For a list of Kafka instance settings that you can update using the CLI, see the `rhoas kafka update` entry in the {base-url-cli}{command-ref-url-cli}[CLI command reference (rhoas)^].
+
+.Prerequisites
+* You have created a Kafka instance. To learn how to do this, see {base-url}{getting-started-url-kafka}[Getting started with {product-long-kafka}^].
+
+.Procedure
+. In the {product-kafka} {service-url-kafka}[web console^], click *Kafka Instances* and select a Kafka instance.
+. Select the *Settings* tab.
+. Adjust any of the settings as needed.
+
+[role="_additional-resources"]
+.Additional resources
+* {base-url}{getting-started-url-kafka}[Getting started with {product-long-kafka}^]
+* {base-url}{getting-started-rhoas-cli-url-kafka}[Getting started with the rhoas CLI for OpenShift Streams for Apache Kafka^]
+* {base-url-cli}{command-ref-url-cli}[CLI command reference (rhoas)^]
+
+
+[id="ref-kafka-instance-settings_{context}"]
+== Kafka instance settings in {product-kafka}
+
+[role="_abstract"]
+You can edit the following Kafka instance settings in {product-long-kafka}.
+
+Connection re-authentication::
```

Comment:
@DuncanDoyle Can you verify that the connection reauth definition is technically accurate? I especially want to make sure that I've stated the security risk correctly.